### PR TITLE
FIX: Action Module Playground & Components Missing Imports

### PR DESCRIPTION
**Pull request by mdelez**
https://dasch.myjetbrains.com/youtrack/issue/DSP-333

### CODE REVIEW

**Comment by tobiasschweizer** [comment]
The warnings when running the test are gone :-)

I still get this when building the lib

> WARNING: No name was provided for external module 'ts-md5' in output.globals – guessing 'tsMd5'

You can simply add an entry to `projects/dsp-ui/ng-package.json`

**Comment by mdelez** [comment]
> The warnings when running the test are gone :-)
> 
> I still get this when building the lib
> 
> > WARNING: No name was provided for external module 'ts-md5' in output.globals – guessing 'tsMd5'
> 
> You can simply add an entry to `projects/dsp-ui/ng-package.json`

Ah, sorry I thought you said you wanted a seperate PR for this in our call but I'm okay doing it in this one.

Fixed in 3c42030 😄

**Comment by tobiasschweizer** [comment]
> Ah, sorry I thought you said you wanted a separate PR for this in our call but I'm okay doing it in this one.

Yes, I said this, sorry. I should be more consistent :-)

File: projects/dsp-ui/ng-package.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
       "@knora/api": "@knora/api",
       "ngx-color-picker": "ngxColorPicker",
       "jdnconvertiblecalendar": "jdnconvertiblecalendar",
-      "jdnconvertiblecalendardateadapter": "jdnconvertiblecalendardateadapter"
+      "jdnconvertiblecalendardateadapter": "jdnconvertiblecalendardateadapter",
+      "ts-md5": "Md5"
```
Comment (tobiasschweizer) [inline projects/dsp-ui/ng-package.json:11]: Shouldn't this be "tsMd5" on the right?

Comment (mdelez) [reply projects/dsp-ui/ng-package.json]: Both seem to work but I changed it to "tsMd5" in 91d6b27

Comment (tobiasschweizer) [reply projects/dsp-ui/ng-package.json]: maybe you can put what you want, I don't know. "tsMd5" is just what the compiler guessed.

**Pull request merged**